moveit_commander/src/moveit_commander/move_group.py

```diff
@@ -596,7 +596,7 @@ def retime_trajectory(self, ref_state_in, traj_in, velocity_scaling_factor=1.0,
         traj_out.deserialize(ser_traj_out)
         return traj_out
 
-    def get_jacobian_matrix(self, joint_values):
+    def get_jacobian_matrix(self, joint_values, reference_point_position=[0.0, 0.0, 0.0]):
```

Comment:
It's bad practice to use mutable default arguments, see https://docs.python-guide.org/writing/gotchas.
Alternatively, use `None` as the default arg but forward the zero list if `None` was given.
```suggestion
    def get_jacobian_matrix(self, joint_values, reference_point_position=None):
```

Reply:
Nice, thanks!